README.md

```diff
@@ -14,3 +14,8 @@ part of the array each recursive call considers.
 Analyse the time complexity of your implementation and give a $\Theta$ bound for
 its worst-case runtime. Add your answer, including your reasoning, to this
 markdown file.
+
+For the time complexity of the code, the outer loop is is moving by two times so the complexity would be O(log n) and the inner loop is accessing and copying each element to the temp array so that is at constant time of O(n). Putting them together we get O(nlogn) and because that is best and worst case we dont need to calculate omega and we are left with big Theta (nlog n).
```

Comment:
So the same as normal mergesort? Aren't you doing more work?

Reply:
I fixed the reasoning but it should be about the same for in place as it is normally.